### ISSUE TITLE
Posibility of user accounts to save/sync data?

### ISSUE THREAD

**Issue by rosstex** [mDibyo/berkeley-scheduler#40]
It would be nice to keep my schedule synced between different devices, and I had to recreate it when my computer crashed as well.

/stretchgoal

**Comment by mDibyo** [comment]
Yes that would indeed by nice! :D It sucks that your computer crashed. :/

One thing that could be pretty easily implemented is a save page. You copy the url of the page, and paste it on a new device, and your courses, and schedules are transferred. But that wouldn't prevent you
from losing your schedules if your computer crashed.

On the current setup, syncing is not a possibility, since there is no server to speak of, on which to store user data (the whole thing is just Github Pages). In future, it would indeed be possible to set this up, maybe stand up a heroku server, and do all sorts of things on it, including accounts, email sending to friends/yourself etc., And I think, we would eventually want that.

So, yeah, adding that to stretch goals. :)

**Comment by rosstex** [comment]
Ah, that makes sense. The save page would be pretty awesome for me, I could just update my bookmark whenever I'm done editing my schedule and my other devices will pick it up.

**Comment by rosstex** [comment]
Bumping because I've had a lot of pain with my data getting wiped!

**Comment by mDibyo** [comment]
https://scotch.io/tutorials/%E2%80%8Bbuild-an-angular-app-with-user-authentication-in-10-minutes

**Comment by mDibyo** [comment]
Update: I am currently working on user accounts in the user-accounts2 branch. It should be done soon!
